mod/likes/views/default/likes/popup.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$guid = get_input("guid");
+
+elgg_entity_gatekeeper($guid);
```

Comment:
that means invalid input is not handled, at least we should provide a system_message in that case